DFReader.py

```diff
@@ -8,10 +8,12 @@
 Partly based on SDLog2Parser by Anton Babushkin
 '''
 from __future__ import print_function
+from past.builtins import long
```

Comment:
A [different solution](https://github.com/ArduPilot/pymavlink/commit/463b63c8339d09a7c7479fe6f383c047450b24df) to line 11 was just checked into master.  The rest of this PR is very important for achieving Python 3 functionality.